devtools/x/Cargo.toml

```diff
@@ -23,3 +23,4 @@ glob = "0.3.0"
 x-core = { version = "0.1.0", path = "../x-core" }
 x-lint = { version = "0.1.0", path = "../x-lint" }
 libra-workspace-hack = { path = "../../common/workspace-hack", version = "0.1.0" }
+clap = "2.33.3"
```

Comment:
I'd recommend using the version of `clap` that structopt re-exports, to minimize version skew issues: https://docs.rs/structopt/0.3.20/structopt/#reexports

Reply:
excellent, and done, didn't know that was a thing.